recipes/libnfnetlink/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.29.1"
+
+class LibnfnetlinkConan(ConanFile):
+    name = "libnfnetlink"
+    license = "GPL-2.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://netfilter.org/projects/libnfnetlink/index.html"
+    description = "low-level library for netfilter related kernel/userspace communication"
+    topics = ("libnfnetlink", "netlink", "netfilter")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("libnfnetlink is only supported on Linux")
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        conf_args = []
+        if self.options.shared:
+            conf_args.extend(["--enable-shared", "--disable-static"])
+        else:
+            conf_args.extend(["--disable-shared", "--enable-static"])
+        self._autotools.configure(configure_dir=self._source_subfolder, args=conf_args)
+        return self._autotools
+
+    def build(self):
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+
+        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "etc"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
You should explicit set the pkg_config name

Reply:
Thanks for reviewing! Where do I set the pkg_config name? in `package_info`? 
Will something like this work: 
> self.cpp_info.names["pkg_config"] = "libnfnetlink"

Reply:
Yes

https://github.com/conan-io/conan-center-index/blob/2b4c8896953ad9c811a1bff4482bdfce8a0cf7bd/recipes/wayland/all/conanfile.py#L82

Reply:
Thanks! Done.